Cargo.toml

```diff
@@ -52,3 +52,14 @@ members = ["data", "irc", "irc/proto"]
 [patch.crates-io]
 iced = { git = "https://github.com/iced-rs/iced", rev = "a05b8044a9a82c1802d4d97f1723e24b9d9dad9c" }
 iced_core = { git = "https://github.com/iced-rs/iced", rev = "a05b8044a9a82c1802d4d97f1723e24b9d9dad9c" }
+
+[profile.packaging]
+opt-level = 3
+debug = false
+lto = "fat"
+strip = "debuginfo"
+panic = 'unwind'
+incremental = false
+codegen-units = 1
+rpath = false
```

Comment:
We don't need to redefine the items that are identical to the `release` profile since we use `inherits` below. Let's remove them.